app/views/publisher_information/publisher_updates.govspeak.md

```diff
@@ -1,3 +1,20 @@
+
+### Email links, content data, and guidance updates <span class="govuk-caption-m">16 May 2019</span>
+Added a link to [Content Data](https://content-data.publishing.service.gov.uk/content) from the edit screen so you can easily check the performance of published content.
```

Comment:
We should time this with some work tomorrow morning to remove the organisation restriction of this. Perhaps @1pretz1 can pair with Jonathon on it.

Reply:
Yup this sounds good, I'll pair with Jonathon when he's free and update here when a pr is ready.